diff --git a/components/content_settings/core/browser/content_settings_pref_provider.cc b/components/content_settings/core/browser/content_settings_pref_provider.cc
index a00cacb48f3b91597edc46926bed38f8ed837c35..921e2d88af867757504b29a6facc8cfc3903c050 100644
--- a/components/content_settings/core/browser/content_settings_pref_provider.cc
+++ b/components/content_settings/core/browser/content_settings_pref_provider.cc
@@ -49,8 +49,6 @@ const char kObsoleteFullscreenExceptionsPref[] =
 #if !defined(OS_ANDROID)
 const char kObsoleteMouseLockExceptionsPref[] =
     "profile.content_settings.exceptions.mouselock";
-const char kObsoletePluginsExceptionsPref[] =
-    "profile.content_settings.exceptions.plugins";
 const char kObsoletePluginsDataExceptionsPref[] =
     "profile.content_settings.exceptions.flash_data";
 #endif  // !defined(OS_ANDROID)
@@ -119,6 +117,7 @@ PrefProvider::PrefProvider(PrefService* prefs,
       clock_(base::DefaultClock::GetInstance()) {
   TRACE_EVENT_BEGIN0("startup", "PrefProvider::PrefProvider");
   DCHECK(prefs_);
+
   // Verify preferences version.
   if (!prefs_->HasPrefPath(prefs::kContentSettingsVersion)) {
     prefs_->SetInteger(prefs::kContentSettingsVersion,
@@ -198,6 +197,7 @@ bool PrefProvider::SetWebsiteSetting(
   // sites/origins defined by the |primary_pattern| and the |secondary_pattern|.
   // Default settings are handled by the |DefaultProvider|.
   if (primary_pattern == ContentSettingsPattern::Wildcard() &&
+      BRAVE_SET_WEBSITE_SETTINGS
       secondary_pattern == ContentSettingsPattern::Wildcard()) {
     return false;
   }
@@ -282,7 +282,6 @@ void PrefProvider::DiscardOrMigrateObsoletePreferences() {
   prefs_->ClearPref(kObsoleteFullscreenExceptionsPref);
 #if !defined(OS_ANDROID)
   prefs_->ClearPref(kObsoleteMouseLockExceptionsPref);
-  prefs_->ClearPref(kObsoletePluginsExceptionsPref);
   prefs_->ClearPref(kObsoletePluginsDataExceptionsPref);
 #endif  // !defined(OS_ANDROID)
 #endif  // !defined(OS_IOS)
