diff --git a/components/content_settings/core/browser/BUILD.gn b/components/content_settings/core/browser/BUILD.gn
index 2ce3bb2ed29a1149d2495f1f12f33fb7805d30cb..efe2daded7e0ccf32bdf41333c1d0f5c6caf6984 100644
--- a/components/content_settings/core/browser/BUILD.gn
+++ b/components/content_settings/core/browser/BUILD.gn
@@ -81,6 +81,7 @@ static_library("browser") {
       "//components/policy/core/browser",
       "//third_party/blink/public/common:headers",
     ]
+    sources += brave_components_content_settings_core_browser_sources deps += brave_components_content_settings_core_browser_deps
   }
 
   configs += [
